.travis.yml

```diff
@@ -29,3 +29,19 @@ addons:
 
 after_success:
 - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+
+deploy:
+  # runs semantic-release on each
+  - provider: script
+    script: npm ci && npm run semantic-release
+    on:
+      tags: false
+      all_branches: true
```

Comment:
Shouldn't branch limit to "master"?

Reply:
It will do a dry run if branch is not master (marked in releaserc)

Reply:
Why we need a dry run?

Reply:
to check that the setup is not broken (ie in case we update some of packages to incompatible versions)

Reply:
Will the dry run works when GITHUB_TOKEN & NPM_TOKEN is absent?